extensions/amp-script/0.1/amp-script.js

```diff
@@ -271,6 +282,15 @@ export class AmpScript extends AMP.BaseElement {
 
     const sandbox = this.element.getAttribute('sandbox') || '';
     const sandboxTokens = sandbox.split(' ').map((s) => s.trim());
+    let iframeUrl;
+    if (getMode().localDev) {
+      const folder = getMode().minified ? 'current-min' : 'current';
+      iframeUrl = `/dist.3p/${folder}/amp-script-proxy-iframe.html`;
+    } else {
+      iframeUrl = `${urls.thirdParty}${
+        getMode().rtvVersion
```

Comment:
I believe 3p uses AMP Version, not RTV. Eg, it's https://3p.ampproject.net/2104022034000/frame.html, not https://3p.ampproject.net/012104022034000/frame.html

Reply:
Nice spot. I'll update